internal/utils/http/http.go

```diff
@@ -170,10 +170,8 @@ func SendErrorResponse(w http.ResponseWriter, r *http.Request, err error) {
 func sendErrorResponse(w http.ResponseWriter, r *http.Request, error rpccode.Code, message string) {
 	logger := logging.FromContext(r.Context())
 
-	errorCode := rpccode.Code_value[error.String()]
-
 	status := errorResponse{
-		Status:  errorCode,
+		Status:  error.String(),
```

Comment:
Have you tested that it returns strings from column RPC here? https://cloud.google.com/apis/design/errors#handling_errors

Reply:
It returns these values: https://godoc.org/google.golang.org/genproto/googleapis/rpc/code
I didn't check it one by one but it looks the same, looking at it.

Reply:
Looks good. Hopefully Google has same error codes in different producs :D